source/copilot-schema.txt

```diff
@@ -0,0 +1,63 @@
+.. _vsce-copilot-schema:
+
+===============
+/schema Command
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The ``/schema`` command provides information about your collection
+schema. The |copilot| accesses your connected MongoDB deployment to
```

Comment:
Consider rewrite in form "To XXX, the YYY."

Reply:
Updated this section to simplify!